pkg/glusterfs/utils/config.go

```diff
@@ -16,7 +16,7 @@ func RoundUpSize(volumeSizeBytes int64, allocationUnitBytes int64) int64 {
 	return (volumeSizeBytes + allocationUnitBytes - 1) / allocationUnitBytes
 }
 
-// RoundUpToGB rounds up given quantity upto chunks of GiB
+// RoundUpToGB rounds up given quantity upto chunks of GB
```

Comment:
+1 Looking at the code above shows the use of Base 10 multipliers. It's very important that we be consistent here or we'll likely run into sizing bugs down the road.